var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -674,7 +706,96 @@ def define_tpl_enable(cmake_var, spec_var=None):
 
         options.append(self.define_from_variant('TPL_ENABLE_ADIOS2', 'adios2'))
 
+        # set up Kokkos CPU architecture option
+        kokkosArchCPU = {
```

Comment:
You should be able to do:
```python
from spack.pkg.builtin.kokkos import Kokkos
```

and use `Kokkos.spack_micro_arch_map` instead. Also note that your symbol here `kokkosArchCPU` is not PEP-8 compliant.